.github/workflows/ci.yml

```diff
@@ -33,3 +35,33 @@ jobs:
     - run: npm test
       env:
         CI: true
+
+  job_2:
+
+    name: Deploy to gh-pages when push to master branch
+
+    needs: job_1
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        if: github.event_name == 'push' && github.ref == 'refs/heads/master'
```

Comment:
Here, master branch will change to main branch from Oct. 1st

Reply:
thanks!